src/components/motion/MotionController.cpp

```diff
@@ -31,13 +31,43 @@ bool MotionController::ShouldWakeUp(bool isSleeping) {
   }
   return false;
 }
+
+bool MotionController::ShouldRollDice() {
+  if (z > 0) {
+    diceRollFirstTrigger = false;
+    return false;
+  }
+
+  if (x > 480 || x < -480) {
+    diceRollFirstTrigger = false;
+    return false;
+  }
+
+  if (y > 550 && not diceRollFirstTrigger) {
+    diceRollFirstTrigger = true;
+    return false;
+  }
+
+  if (y < -240 && diceRollFirstTrigger) {
+    diceRollFirstTrigger = false;
+    return true;
+  }
+
+  return false;
+}
 void MotionController::IsSensorOk(bool isOk) {
```

Comment:
Maybe add line breaks here?

Reply:
I guess I'm not sure what you mean -- these lines don't look too long to me.  They fit with the clang-format, at least.